O365/message.py

```diff
@@ -756,6 +762,24 @@ def reply(self, to_all=True):
 
         message = response.json()
 
+        # adding new feacture to the timezone 
+        html_content = message['body']['content']
```

Comment:
Variable name `html_content` could be more descriptive, something like `message_body_content_html` makes it instantly obvious what it contains and builds upon previously-defined `message` variable naming.